IS7024FinalProjectBreweryArc/BreweryArc/Pages/Index.cshtml

```diff
@@ -45,7 +47,7 @@
             @{
                 foreach (Breweries.BreweryCollection brr in brew)
                 {
-                    <tr>
+                    <tr style="background-color:ButtonHighlight">

```

Comment:
From the user preceptive, For the table section I have highlighted the rows and columns for better UI experience.
https://www.w3schools.com/cssref/pr_background-color.asp

Reply:
Inline styles are not permitted in XHTML 1.0 Strict.  Inline styles cause bloat, and tie content to presentation.  Use linked styles instead.